Sources/SnapshotTestingExtensions/UITraitCollection.swift

```diff
@@ -0,0 +1,114 @@
+        let base: [UITraitCollection] = [
+            UITraitCollection(forceTouchCapability: .available),
+            UITraitCollection(layoutDirection: .leftToRight),
+            UITraitCollection(preferredContentSizeCategory: .medium),
+            UITraitCollection(userInterfaceIdiom: .phone)
+        ]
+        return switch orientation {
+        case .landscape:
+            UITraitCollection(
+                traitsFrom: base + [
+                    UITraitCollection(horizontalSizeClass: .regular),
+                    UITraitCollection(verticalSizeClass: .compact)
+                ]
+            )
+        case .portrait:
+            UITraitCollection(
+                traitsFrom: base + [
+                    UITraitCollection(horizontalSizeClass: .compact),
+                    UITraitCollection(verticalSizeClass: .regular)
+                ]
+            )
+        }
+    }
+
+    internal static func iPhone14Pro(_ orientation: ViewImageConfig.Orientation) -> UITraitCollection {
+        let base: [UITraitCollection] = [
+            UITraitCollection(forceTouchCapability: .available),
+            UITraitCollection(layoutDirection: .leftToRight),
+            UITraitCollection(preferredContentSizeCategory: .medium),
+            UITraitCollection(userInterfaceIdiom: .phone)
+        ]
+        return switch orientation {
+        case .landscape:
+            UITraitCollection(
+                traitsFrom: base + [
+                    UITraitCollection(horizontalSizeClass: .regular),
+                    UITraitCollection(verticalSizeClass: .compact)
+                ]
+            )
+        case .portrait:
+            UITraitCollection(
+                traitsFrom: base + [
+                    UITraitCollection(horizontalSizeClass: .compact),
+                    UITraitCollection(verticalSizeClass: .regular)
+                ]
+            )
+        }
+    }
+
+    internal static func iPhone14ProMax(_ orientation: ViewImageConfig.Orientation) -> UITraitCollection {
+        let base: [UITraitCollection] = [
+            UITraitCollection(forceTouchCapability: .available),
+            UITraitCollection(layoutDirection: .leftToRight),
+            UITraitCollection(preferredContentSizeCategory: .medium),
+            UITraitCollection(userInterfaceIdiom: .phone)
+        ]
+        return switch orientation {
+        case .landscape:
+            UITraitCollection(
+                traitsFrom: base + [
+                    UITraitCollection(horizontalSizeClass: .regular),
+                    UITraitCollection(verticalSizeClass: .compact)
+                ]
+            )
+        case .portrait:
+            UITraitCollection(
+                traitsFrom: base + [
+                    UITraitCollection(horizontalSizeClass: .compact),
+                    UITraitCollection(verticalSizeClass: .regular)
+                ]
+            )
```

Comment:
May we try this alternate style?
```suggestion
            UITraitCollection(traitsFrom: base + [
                UITraitCollection(horizontalSizeClass: .compact),
                UITraitCollection(verticalSizeClass: .regular)
            ])
```